components/camel-hdfs/src/main/java/org/apache/camel/component/hdfs/HdfsFileType.java

```diff
@@ -54,9 +41,21 @@
 import org.apache.hadoop.io.Text;
 import org.apache.hadoop.io.Writable;
 import org.apache.hadoop.io.WritableComparable;
-import org.apache.hadoop.util.Progressable;
 import org.apache.hadoop.util.ReflectionUtils;
 
+import java.io.ByteArrayOutputStream;
```

Comment:
This breaks the code style. Please run the module build with sourcecheck profile enabled and check the warnings

Reply:
Is there a codestype profile for intelij I could use? I've done the formatting with the default setting.

Reply:
The available stuff is here https://github.com/apache/camel/tree/master/buildingtools/src/main/resources